homeworks/misha.kolosov_mwkolosov/homework_7/main.js

```diff
@@ -0,0 +1,16 @@
+const random = Math.floor(Math.random() * (100 - 50 + 1)) + 50;
+
+const fib = (iter) => {
```

Comment:
it seems like more logical approach is to pass not a number of iterations, but maximum fibo number value

you need to refactor your loop